recipes/recipes_emscripten/xeus_shared/recipe.yaml

```diff
@@ -0,0 +1,39 @@
+context:
+  version: 5.2.2
+
+package:
+  name: xeus
```

Comment:
Why do we need a different recipe from the xeus main xeus one?

Note: this sems to be naming the package `xeus` anyways.